README.md

```diff
@@ -47,45 +47,24 @@ syntax [here][bazel-cli]. Common [commands][bazel-cli-commands] are
 
 ### Nixpkgs
 
-This rule set supports [Nixpkgs][nixpkgs]. If you are on NixOS, or if
-you are using Nixpkgs on your project, consider passing the following
-argument on the command-line to select a Nixpkgs-based toolchain for
-the build:
+This rule set supports using [Nixpkgs][nixpkgs] to provision your GHC
+toolchain and to fetch hackage packages from there. To create your
+project, pass `--use-nix`, like so:
 
-```
-$ bazel build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs //...
+```console
+$ sh <(curl https://haskell.build/start) --use-nix
 ```
 
-See [below](#saving-common-command-line-flags-to-a-file) to
-permanently set that flag.
+This generates the same files as above, but uses `nixpkgs` to
+provision GHC.
+
+If you are on NixOS, this is the only way to set up your project,
```

Comment:
See @bethaitman's Haskell documentation talk: don't put important information (like something being required for NixOS) at the end of the section. Start with that.

Reply:
Added a sentence at the top of `Setup`.